### PR TITLE
[feat]:Adding css for badges

### PR DESCRIPTION
**Pull request by raghav2110**
Hi , I have added css for badges. Please review my code.
I have also made a change in avatar as it was required for badge
Screenshot 2022-01-28 at 9 33 58 PM
.

### CODE REVIEW

File: components/badges/badges.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+.badge-stat{
+    color: green;
+    position: absolute;
+    right: 0.1rem;
+    bottom: 0.1rem;
+}
+
+.badge-stat-small{
+    font-size: smaller;
```
Comment (junaidshaikh-js) [inline components/badges/badges.css]: Check what value this keywords hold and try to do it in "rem" if possible

Comment (raghav2110) [reply components/badges/badges.css]: Made changes in pull request. Changes to rem. Please review it once.

File: components/badges/badges.css [hunk 2]
```diff
@@ -22,11 +22,11 @@
 }
 
 .badge-stat-small{
-    font-size: smaller;
+    font-size: 0.5rem;
 }
 
 .badge-stat-medium{
-    font-size: small;
+    font-size: 0.8rem;
```
Comment (saadahmadhere) [inline components/badges/badges.css:29]: Good that you changed it to variable size.

**Review by junaidshaikh-js — COMMENTED** [review]
You made the required changes. We are good to proceed.

**Review by saadahmadhere — COMMENTED** [review]
Looks great after the required changes. Good to proceed.

**Pull request merged**